docs/src/app/components/master.jsx

```diff
@@ -21,26 +21,22 @@ const ThemeManager = Styles.ThemeManager;
 const DefaultRawTheme = Styles.LightRawTheme;
 
 
-const Master = React.createClass({
-  getInitialState () {
-    return {
+class Master extends React.Component {
+  constructor(props, context) {
+    super(props, context);
+    this._handleTabChange = this._handleTabChange.bind(this);
```

Comment:
ES2015 class components don't use autobinding like React.createClass does. So you need to bind event handlers manually. If you want to use @autobind or whatever other solution, the codemod can be adjusted and re-run to use that. Another solution we are hoping to make part of the ES2015 spec is to be able to define arrow functions as class properties, like this:

```js
class A {
  onClick: () => this.foo();
} 
```

which would avoid the need for calling bind. But this hasn't been added to the next version of ES *yet*.